Tests/NavigationTests/DistributedNavigationDelegateTests.swift

```diff
@@ -1667,3 +1665,11 @@ class DistributedNavigationDelegateTests: DistributedNavigationDelegateTestsBase
     }
 
 }
+
+// swiftlint:enable line_length
```

Comment:
Some of these should be unnecessary now because of the disabled rules right? Dunno if it's worth fixing here though

Reply:
Absolutely they should - in fact SwiftLint will warn you when you have a disabled rule that didn't actually stop any warnings from popping up. I think I fixed these in PR 3, but let me try and clean them up in here if I can.

Reply:
Done ✅